install-deps.sh

```diff
@@ -28,6 +28,9 @@ function in_jenkins() {
     test -n "$JENKINS_HOME"
 }
 
+source src/script/pypackages.sh
+install-pip-packages
```

Comment:
I'd put this closer to (or inside) the `preload_wheels_for_tox()` or the chunk that installs Python stuff, which also will probably need a fix after these changes.